lib/constant.dart

```diff
@@ -0,0 +1,3 @@
+import 'package:flutter/material.dart';
```

Comment:
file name doesn't match his content 
make a class for constant colors  and class for constant String